src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+class Bulldozer extends Machine {

```

Comment:
The class `Bulldozer` should have an access modifier. Consider using the `public` modifier to follow the common convention for classes which are likely to be used from other packages.